src/components/DisplayComment.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import Proptypes from 'prop-types';
+
+import { instance } from '../utils/config';
+
+/**
+ *
+ *
+ * @class DisplayComment
+ * @extends {React.Component}
+ */
+class DisplayComment extends React.Component {
```

Comment:
Rename `DisplayComment` to `CommentWrapper` or something. Use verb for defining function, for defining variable or class name it should be noun.